packages/bff/.env

```
@@ -36,3 +37,6 @@ AWS_CONFIG_FILE=aws.config.local
 
 SELFCARE_V2_URL=localhost
 SELFCARE_V2_API_KEY=key
+
+ESERVICE_DOCUMENTS_PATH="backend-for-frontend.storage.eservice-documents.path"
```

We should use the same value used in catalog process

```suggestion
ESERVICE_DOCUMENTS_PATH="interop-eservice-documents"
```